### PR TITLE
update README with examples

### LINKED ISSUE #7: Adding documentation with example
It would be nice to have some documentation (e.g. screen capture gif, markdown) of an example usage (e.g. with FOAMS stimuli)

### PR DESCRIPTION
**Pull request by hansen508**
Okay, admittedly this is kind of an eyesore with probably too much file-embedding, but I've added some additional documentation with examples! 

Couple specific questions I'd like your feedback on:

1. How to reduce image size onscreen? They def don't need to take up the full width. I haven't been able to figure out an easy way to do this through code (at least in restructured text), but can keep googling or manipulate outside of Git otherwise
2. Where best to store demo screenshots/gifs? Right now I plopped them into ./docs/media/, but please move them if they make more sense elsewhere
3. Are there too many examples, or images that are unnecessary? Or, alternatively, other things you'd like a depiction of?

### CODE REVIEW

**Comment by hansen508** [comment]
Okay, I think I fixed the thing that was causing the tests to fail (and merged it in another pull request). Is it possible to rerun the tests? @danibene

File: README.rst [hunk 1]
```diff
@@ -29,6 +29,124 @@ MisoSOUPy
...
+
+.. figure:: ./docs/media/refinement1.png
+
+*Example: Instructions for refining sound selections, ranking the top 3.*
+
+.. figure:: ./docs/media/5_MisoSoupy_DemoVid.gif
+
+*Example: Ranking trigger sound selections from 1 (more triggering) to 3 (less triggering).*
+
+**When more sounds are selected than need are needed for an experiment, not all sounds will be ranked. Refining the top sounds helps the researcher keep the number of sound choices consistent across participants.**
```
Comment (danibene) [inline README.rst]:
```suggestion
**When more sounds are selected than are needed for an experiment, not all sounds will be ranked. Refining the top sounds helps the researcher keep the number of sound choices consistent across participants.** 
```

File: README.rst [hunk 2]
```diff
@@ -29,6 +29,124 @@ MisoSOUPy
...
+
+*Example: Selecting neutral sounds from the remaining items, then ranking top 3.*
+
+**When ranking sounds, choices may be reset and changed if necessary.**
+
+.. figure:: ./docs/media/3_MisoSoupy_DemoVid.gif
+
+*Example: Ranking top 5 neutral sounds, utilizing the reset button to make changes.*
+
+**After selecting and ranking sounds, the participant's choices will be output to a tab-delimited .txt file.**
```
Comment (danibene) [inline README.rst]:
```suggestion
**After selecting and ranking sounds, the participant's choices will be saved to a tab-delimited .txt file.**
```

File: README.rst [hunk 3]
```diff
@@ -29,6 +29,124 @@ MisoSOUPy
...
+
+
+Setup
+=====
+
+To use MisoSOUPy, open and run ``run_misosoupy.py`` 
+
+By default, MisoSOUPy will request participants select and rank their top 5 trigger and neutral sounds. To change these default settings, edit ``config.ini``
+
+*Example: change ``step_select_neutral`` to ``False`` in ``config.ini`` to only have participants select triggers sounds.*
```
Comment (danibene) [inline README.rst]:
```suggestion
*Example: change `step_select_neutral` to `False` in `config.ini` to only have participants select trigger sounds.*
```

File: README.rst [hunk 4]
```diff
@@ -29,6 +29,124 @@ MisoSOUPy
...
+   
+   # Request participants select their least triggering (or neutral) sounds. If triggering sounds are selected first, these options will remain in the list but appear grayed out. Step_select_sound_list must be True for this option to be True. (Default = True)
```
Comment (danibene) [inline README.rst]:
```suggestion
   # Request participants to select their least triggering (or neutral) sounds. If triggering sounds are selected first, these options will remain in the list but appear grayed out. Step_select_sound_list must be True for this option to be True. (Default = True)
```

File: README.rst [hunk 5]
```diff
@@ -29,6 +29,124 @@ MisoSOUPy
...
+   ``step_select_neutral = True``
+
+*Example: change the value for ``num_items_to_select`` in ``config.ini`` to match how many stimuli per category are needed for the experiment.*
```
Comment (danibene) [inline README.rst]:
```suggestion
*Example: change the value for `num_items_to_select` in `config.ini` to match how many stimuli per category are needed for the experiment.*
```

File: README.rst [hunk 6]
```diff
@@ -29,6 +29,124 @@ MisoSOUPy
...
+   
+   # Minimum number of sound labels participants must select in each sound category. If fewer labels than this number are selected, participants see an error screen and must restart. If step_refine_sound_list = True, participants will also rank order this number of sounds. Default = 5.
+  ``num_items_to_select = 5``
+
+Put a folder with your sound files (or a .csv of the sound names, see ``FOAMS_sound_list.csv`` for an example) in the ``/misosoupy/assets/`` directory. Sound labels will be derived from the file names (or .csv) in this folder for presentation by MisoSOUPy.
```
Comment (danibene) [inline README.rst]:
```suggestion
Put a folder with your sound files (or a .csv of the sound names, see `FOAMS_sound_list.csv` for an example) in the `/misosoupy/assets/` directory. Sound labels will be derived from the file names (or .csv) in this folder for presentation by MisoSOUPy.
```

**Comment by danibene** [comment]
> Okay, admittedly this is kind of an eyesore with probably too much file-embedding, but I've added some additional documentation with examples!
> 
> Couple specific questions I'd like your feedback on:
> 
> 1. How to reduce image size onscreen? They def don't need to take up the full width. I haven't been able to figure out an easy way to do this through code (at least in restructured text), but can keep googling or manipulate outside of Git otherwise

I think you could try specifying the width e.g. `:width: 500px` without the scale.

> 2. Where best to store demo screenshots/gifs? Right now I plopped them into ./docs/media/, but please move them if they make more sense elsewhere

Seems like a good place!

> 3. Are there too many examples, or images that are unnecessary? Or, alternatively, other things you'd like a depiction of?

I think the GIFs are awesome! I wonder if we should give more specific installation instructions. Do we want to put this on PyPi?

**Comment by danibene** [comment]
> Okay, I think I fixed the thing that was causing the tests to fail (and merged it in another pull request). Is it possible to rerun the tests? @danibene

You did this on the `add/documentation` branch which you then merged to `dev`, so I ran `git merge dev` to merge those changes into the `update/README` branch

**Pull request merged**